components/CreateGiftCardsForm.js

```diff
@@ -124,10 +123,9 @@ const DeliverTypeRadioSelector = styled(Flex)`
   border-bottom: 1px solid ${themeGet('colors.black.200')};
 `;
 
-const RadioButtonContainer = styled.label`
+const RadioButtonContainer = styled(Strong)`
```

Comment:
Wrapping the `input` in a `label` seems valid to me. What was the problem here?

Reply:
Here the problem is, 

```
A form label must be associated with a control  styled-components-a11y/label-has-associated-control
```

Which points to, https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-associated-control.md. 

My thinking here is that we are using `label` here just to get the default label formatting (kinda bold text and stuff) without necessarily tying that with an input. The formatting itself can be achieved using `Strong`.

Reply:
`label` should indeed not be used for styling, but that wasn't the case here. Putting the `input` inside the `label` is valid HTML and should not require setting `htmlFor`. I think this is a false positive.

Reply:
@Betree : I've corrected this. This shouldn't have been changed at all; it's a mistake on my part I think. 🤔